nwplus/assessment-portal
# Finish assesment basics and bug fixes

This PR reworks a ton of stuff, check out the commit messages for more information.

### Main changes:
1. Reworked how applicants are loaded so that info is always up to date
2. Reworked how scoring works so that it will update from firebase and so that it saves the last user who made a scoring update
3. Reworked how searching/filtering works so that it'll work with the auto updating hackers
4. Setup the auth routes so that they are properly protecting what they should be 
5. Added all the missing fields to the applicant overview
6. Styled the applicant overview a little bit
7. Added a resume link and get the resume from firebase
8. A bunch of minor fixes/finishing incomplete sections (a lot of the overview section still had hard coded data)
### How to test
1. Login
2. Use my account (ciuffiyogi@gmail.com) to test it (as its a complete app, although some of the answers are joke answers).
3. make sure that you can view all fields
4. make sure that scoring works and total score updates
5. make sure that you can open my resume (just some random paper I wrote for school)
